packages/node/src/node-hid-wrapper.ts

```diff
@@ -18,9 +21,13 @@ export class NodeHIDDevice extends EventEmitter implements HIDDevice {
 	}
 
 	public write(data: number[]): void {
-		this.device.write(data).catch((err) => {
-			this.emit('error', err)
-		})
+		this.writeQueue
+			.add(async () => {
+				await this.device.write(data)
+			})
```

Comment:
small nit: you could directly return the promise here

```suggestion
			.add(() => this.device.write(data))
```